support/ebpf/Makefile

```
@@ -62,10 +63,8 @@ arm64:
 errors.h: ../../tools/errors-codegen/errors.json
 	go run ../../tools/errors-codegen/main.go bpf $@
 
-%.ebpf.c: errors.h ;
-
 %.ebpf.$(TARGET_ARCH).o: %.ebpf.c
-	$(BPF_CLANG) $(FLAGS) -o $@
+	$(BPF_CLANG) $(FLAGS) -MMD -MF $(@:.o=.d) -o $@
```

I don't think we need `MF` but we do need `MP`, as header files can be deleted and break the build (until `make clean` is called).

```suggestion
	$(BPF_CLANG) $(FLAGS) -MMD -MF $(@:.o=.d) -o $@
```